queries.js

```diff
@@ -9,34 +9,35 @@ const pool = new Pool({
 
 //Get the list of users
 const getUsers = (request, response) => {
-    pool.query('SELECT * FROM public.users', (error, results) => {
+    pool.query('SELECT * FROM public.users', (error, result) => {
         if(error) {
             throw error;
         }
-        response.status(200).json(results.rows);
+        response.status(200).json(result.rows);
     })
 }
 
 //Get the name of user by ID
 const getUserByID = (request, response) => {
     const id = request.params.id;
-    pool.query('SELECT * FROM users WHERE id = $1', [id], (error, results) => {
+    pool.query('SELECT * FROM users WHERE id = $1', [id], (error, result) => {
         if(error) {
             throw error;
         }
-        response.status(200).json(results.rows);
+        response.status(200).json(result.rows);
     })
 }
 
 //Create a new user
 const createUser = (request, response) => {
     const {name, gender, cmnd, email, birthdate, signupdate, expirydate} = request.body;
-    pool.query('INSERT INTO users(name, gender, cmnd, email, birthdate, signupdate, expirydate) VALUES ($1, $2, $3, $4, $5, $6, $7)', 
-    [name, gender, cmnd, email, birthdate, signupdate, expirydate], (error, result) => {
+    pool.query('SELECT public."InsertUser"($1, $2, $3, $4, $5, $6, $7) as new_id', [birthdate, signupdate, name, cmnd, email, gender, expirydate],
+        //'INSERT INTO users(name, gender, cmnd, email, birthdate, signupdate, expirydate) VALUES ($1, $2, $3, $4, $5, $6, $7)', 
+    (error, result) => {
         if(error) {
             throw error;
```

Comment:
Remove "throw" code so it won't stop the app.  Should console.error() the error then return this json response:
{
  "State" : "Error",
  "ErrorMessage": "Error occurs when execute query on database"
}

Do the same for all error handling code.

Reply:
I fixed it